test/utils.py

```diff
@@ -269,7 +273,11 @@
             "src_pol": 1
         }
     },
-    "config": {}
+    "config": {
+        "develop": {
+            "data_timeout": 1.0
+        }
+    }
```

Comment:
Do you actually need this change? I'd rather have most tests not specify 'develop', so that tests are more similar to real-world conditions where possible.

Reply:
Your "reversion" has now removed the "config" attribute entirely.